std_msgs/Header header

int8 x		    # x-axis [%]
int8 y		    # y-axis [%]
int8 z		    # z-axis [%]
int8 yaw 	    # yaw-axis [%]
int8 camera	  # camera-axis [%]
int8 zoom	    # zoom-axis [%]
byte buttons 	# buttons bitfields (RTL | takeoff/land | back left | back right) {0,1}
